test/verify/check-system-terminal

```diff
@@ -120,6 +120,10 @@ PROMPT_COMMAND='printf "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/
         wait_line(n, prompt)
         b.wait_not_in_text(".terminal .xterm-accessibility-tree", "disconnected")
 
+        # Check that closing the terminal tab is guarded by a confirmation dialog
+        b.close()
+        b.handle_alert(accept=False)
+
```

Comment:
Also, this test should now check that the terminal is still alive after rejecting the confirmation, and that the session indeed gets closed after accepting it.